plugins/SitesManager/lang/en.json

```diff
@@ -112,7 +112,7 @@
         "SiteWithoutDataDetectedSite": "We've detected you're using a %1$s site. Simply follow the instructions from this %2$sguide%3$s.",
         "SiteWithoutDataDetectedSiteWordpress": "We've detected you're using a Wordpress site. Simply %1$sfollow the instructions from this guide%2$s to install the %3$sWP-Matomo plugin%4$s.",
         "SiteWithoutDataDetectedSiteWordpress2": "Set up the WordPress plugin already and not seeing any data? Please check %1$sthis FAQ for help%2$s.",
-        "SiteWithoutDataDetectedGtm": "We've also detected you're using a Google Tag Manager. If you prefer to set up Matomo using GTM please follow this %2$sguide%3$s.",
+        "SiteWithoutDataDetectedGtm": "We've also detected you're using a Google Tag Manager. If you prefer to set up Matomo using GTM please follow this %1$sguide%2$s.",
```

Comment:
Should this be a new translation key? The changed placeholders could cause confusion.